docs/release-notes/v0.44.0.md

```diff
@@ -0,0 +1,59 @@
+# Announcing Radius v0.44.0
+
+Today we're happy to announce the release of Radius v0.44.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+<!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->
+
+## Breaking changes
+
+<!-- ADD ANY BREAKING CHANGES HERE, IF ANY -->
```

Comment:
We should add the `rad resource expose` and `rad resource logs` breaking changes here